### PR TITLE
docs: reorder imports in tutorials 🎨

### PR DESCRIPTION
**Pull request by meysam81**
related #18950 

This PR adds reorders the imports and removes the redundant ones in the [tutorials page](https://airflow.apache.org/docs/apache-airflow/stable/tutorial.html#testing).

### CODE REVIEW

**Review by uranusjr — APPROVED** [review]

File: docs/apache-airflow/tutorial.rst [hunk 1]
```diff
@@ -415,10 +415,9 @@ Let's break this down into 2 steps: get data & merge data:
 
 .. code-block:: python
 
-  from airflow.decorators import dag, task
-  from airflow.hooks.postgres import PostgresHook
-  from datetime import datetime, timedelta
   import requests
+  from airflow.decorators import task
+  from airflow.hooks.postgres_hook import PostgresHook
```
Comment (kaxil) [inline docs/apache-airflow/tutorial.rst]:
```suggestion
  from airflow.hooks.postgres import PostgresHook
```

File: docs/apache-airflow/tutorial.rst [hunk 2]
```diff
@@ -447,6 +446,10 @@ Here we are passing a ``GET`` request to get the data from the URL and save it i
 
 .. code-block:: python
 
+  from airflow.decorators import task
+  from airflow.hooks.postgres_hook import PostgresHook
```
Comment (kaxil) [inline docs/apache-airflow/tutorial.rst]:
```suggestion
  from airflow.hooks.postgres import PostgresHook
```

File: docs/apache-airflow/tutorial.rst [hunk 3]
```diff
@@ -475,10 +478,11 @@ Lets look at our DAG:
 
 .. code-block:: python
 
-  from airflow.decorators import dag, task
-  from airflow.hooks.postgres_hook import PostgresHook
   from datetime import datetime, timedelta
+
   import requests
+  from airflow.decorators import dag, task
+  from airflow.hooks.postgres_hook import PostgresHook
```
Comment (kaxil) [inline docs/apache-airflow/tutorial.rst]:
```suggestion
  from airflow.hooks.postgres import PostgresHook
```

**Review by kaxil — COMMENTED** [review]
The PR and commit description needs to change. Based on the change the imports are not missing anymore, they are just being reordered and in one diff a redundant import is being removed

**Review by kaxil — APPROVED** [review]

**Pull request merged**